src/node.js

```diff
@@ -393,15 +399,22 @@
     function TickObject(c) {
       this.callback = c;
       this.domain = process.domain || null;
+      this._args = undefined;
     }
 
+    TickObject.prototype._setArgs = function _setArgs() {
+      this._args = arguments;
```

Comment:
```js
var i = arguments.length;
var args = new Array(i);
while (i--) args[i] = arguments[i];
```

(i have minimal versions of both slice and splice haha) https://gist.github.com/Fishrock123/98c35a0c745cb59d7496